ob-elasticsearch.el

```diff
@@ -115,7 +119,9 @@ to do that."
     (beginning-of-buffer)
     (es-mark-request-body)
     (let ((output (when mark-active
-                    (es-org-execute-request (cdr (assoc :jq params))))))
+                    (es-org-execute-request
+                     (cdr (assoc :jq params))
+                     (cdr (assoc :tablify params))))))
```

Comment:
We don't need our own flag here. You can use `:results table` instead. See http://orgmode.org/manual/results.html for more info.